included/10.1.0/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# WARNING: this file was autogenerated by generate-included-image.js
+# using
+#   npm run add:included -- 10.1.0 cypress/browsers:node16.14.2-slim-chrome100-ff99-edge
+#
+# build this image with command
+#   docker build -t cypress/included:10.1.0 .
+#
+FROM cypress/browsers:node16.14.2-slim-chrome100-ff99-edge
```

Comment:
Should this be [node16.5.0-chrome97-ff96](https://github.com/cypress-io/cypress-docker-images/tree/master/browsers/node16.5.0-chrome97-ff96)?